src/ResourceManagement/AnalysisServices/AnalysisServices.Tests/ScenarioTests/ServerOperationsTests.cs

```diff
@@ -53,7 +53,7 @@ public void CreateGetUpdateDeleteTest()
                     Console.WriteLine(ex.ToString());
                 }
 
-                Assert.Equal(resultCreate.ProvisioningState, "Succeeded");
+                Assert.Equal(resultCreate.State, "Succeeded");
```

Comment:
Here and all other places. Keep the ProvisioningState check for Succeeded or Failed.